Axe is not working locally

Error message below:
```
➜  web-design-standards-docs git:(add-utility-docs) npm test

> @ test /Users/mayaben-ari/Code/web-design-standards-docs
> npm run axe && npm run lint && npm run crawl && bundle exec rspec


> @ axe /Users/mayaben-ari/Code/web-design-standards-docs
> node config/run-axe.js

Static file server is listening at http://f09r9j-2jmg3qt:57877.
Chrome is debuggable on http://localhost:57878.
Running aXe on:
  /page-templates/docs/ A network request failed to load.
{ requestId: '51F129798E6A0124B92DFDA7559CC0E1',
  timestamp: 79243.403367,
  type: 'Document',
  errorText: 'net::ERR_NAME_NOT_RESOLVED',
  canceled: false }
Terminating with exit code 1.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @ axe: `node config/run-axe.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @ axe script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/mayaben-ari/.npm/_logs/2018-05-24T23_24_10_766Z-debug.log
npm ERR! Test failed.  See above for more details.
```

--- a/package.json
+++ b/package.json
@@ -52,7 +52,6 @@
     "chalk": "^2.4.1",
     "cheerio": "^1.0.0-rc.2",
     "chrome-launcher": "^0.10.2",
-    "chrome-remote-interface": "^0.25.5",
     "del": "^3.0.0",
     "express": "^4.16.2",
     "gulp": "^4.0.0",
@@ -98,5 +97,8 @@
       "build-js"
     ]
   },
-  "snyk": true
+  "snyk": true,
+  "devDependencies": {
+    "chrome-remote-interface": "^0.25.6"
+  }
 }

--- a/package.json
+++ b/package.json
@@ -52,6 +52,7 @@
     "chalk": "^2.4.1",
     "cheerio": "^1.0.0-rc.2",
     "chrome-launcher": "^0.10.2",
+    "chrome-remote-interface": "^0.25.6",
     "del": "^3.0.0",
     "express": "^4.16.2",
     "gulp": "^4.0.0",
@@ -97,8 +98,5 @@
       "build-js"
     ]
   },
-  "snyk": true,
-  "devDependencies": {
-    "chrome-remote-interface": "^0.25.6"
-  }
+  "snyk": true
 }

